src/main/java/seedu/storemando/ui/MainWindow.java

```diff
@@ -181,6 +191,10 @@ private CommandResult executeCommand(String commandText) throws CommandException
             CommandResult commandResult = logic.execute(commandText);
             logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
+
+            //Refresh page
+            tablePanel = new TablePanel(logic.getFilteredItemList());
```

Comment:
It would be better if we create a method like "refreshExpiringItemsPanel()" to adhere to the SLAP principle in this method.

Reply:
All right will do! I will create the same method as you.